wear/src/main/java/io/homeassistant/companion/android/tiles/TileActionReceiver.kt

```diff
@@ -28,16 +30,27 @@ class TileActionReceiver : BroadcastReceiver() {
         val entityId: String? = intent?.getStringExtra("entity_id")
 
         if (entityId != null) {
-            runBlocking {
-                if (wearPrefsRepository.getWearHapticFeedback() && context != null) hapticClick(context)
-
-                try {
-                    onEntityPressedWithoutState(
-                        entityId = entityId,
-                        integrationRepository = serverManager.integrationRepository()
-                    )
-                } catch (e: Exception) {
-                    Log.e(TAG, "Cannot call tile service", e)
+            if (entityId.split(".")[0] == "app_shortcut" && context != null) {
+                val m = mapOf(
+                    "assist" to ConversationActivity.newInstance(context),
+                    "home_assistant" to HomeActivity.newInstance(context)
+                )
+                m[entityId.split(".")[1]]?.let {
+                    it.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+                    context.startActivity(it)
+                }
+            } else {
+                runBlocking {
+                    if (wearPrefsRepository.getWearHapticFeedback() && context != null) hapticClick(context)
```

Comment:
I think users may expect to continue to get feedback on clicks in the tile, I think we have added it in other places as well

Reply:
Adding the haptic feedback when the icon is clicked?

Sure that's easily added. However for clicking on entity shortcuts there is no inherent feedback other than the added haptics. These 2 additions would open an app on screen, which is likely feedback enough by itself.

Reply:
yea in the past we have gotten requests from users for things not related to entity action we can leave it out for now and add it back upon request